src/test/java/com/meilisearch/integration/SettingsTest.java

```diff
@@ -844,6 +845,51 @@ public void testUpdateDistinctAttributeSettingsUsingNull() throws Exception {
         assertEquals(initialDistinctAttribute, resetDistinctAttribute);
     }
 
+    /** Tests of the pagination setting methods */
+    @Test
+    @DisplayName("Test get pagination settings by uid")
+    public void testGetPaginationSettings() throws Exception {
+        Index index = createIndex("testGetPaginationSettings");
+        Settings initialSettings = index.getSettings();
+        Pagination initialPagination = index.getPaginationSettings();
+
+        assertEquals(initialSettings.getPagination().getMaxTotalHits(), 1000);
+        assertNotNull(initialPagination.getMaxTotalHits());
+    }
+
+    @Test
+    @DisplayName("Test update pagination settings")
+    public void testUpdatePaginationSettings() throws Exception {
+        Index index = createIndex("testUpdatePaginationSettings");
+        Pagination newPagination = new Pagination();
+
+        Integer MaxTotalHitsTypos = 100;
+
+        newPagination.setMaxTotalHits(MaxTotalHitsTypos);
+        index.waitForTask(index.updatePaginationSettings(newPagination).getTaskUid());
+        Pagination updatedPagination = index.getPaginationSettings();
+
+        assertTrue(updatedPagination.getMaxTotalHits() == 100);
```

Comment:
```suggestion
        assertEquals(100, updatedPagination.getMaxTotalHits());
```
Also, I realize that it is more complex to debug when you use `assertTrue` because the error was:
```
org.opentest4j.AssertionFailedError: expected: <true> but was: <false>
```
Instead of this, when we used `assertEquals`:
```
expected: <0> but was: <100>
```
So I suggest you change it, WDYT?

Reply:
Oh you are right, I didn't think about that when I was working on it. Let me change it!